configs/Base-RCNN-region-c4.yaml

```diff
@@ -0,0 +1,31 @@
+MODEL:
```

Comment:
For consistency we can either use "region" for all the files, or "updn" in all the files.